### PR TITLE
Adding 4.3 additions to the changelog

### PR DESCRIPTION
**Pull request by wmvanvliet**
This are all the changes I could find since the 4.2.1 release. #199

### CODE REVIEW

**Comment by ccordoba12** [comment]
@willingc, please help us to review this one

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -14,7 +14,29 @@ Changes in Jupyter Qt console
...
+- fix regression where prompt would never be shown for --existing consoles
+- fix "python.exe -m qtconsole" on windows
+- fix showing error messages when running a script using %run
+- fix "invalid cursor position" error and subsequent freezing of user input
+- fix syntax coloring when attaching to non-ipython kernels
+- fix printing when using QT5
+- fix Ctrl+K shortcut (delete until end of line) on OSX
+- fix history browsing (Up/Down keys) when lines are longer than the terminal width
+- fix saving HTML with inline PNG for Python 3 
+- various internal bugfixes
```
Comment (ccordoba12) [inline docs/source/changelog.rst]: Please capitalize each new entry in this list, i.e.

`various internal ...` -> `Various internal ...`

etc.

**Comment by willingc** [comment]
Let's do this... start with a few bullets major changes, followed by all the "Adds", "Changes", and "Fixs" grouped together. This will help the users :-)

**Comment by willingc** [comment]
@wmvanvliet @ccordoba12 I'm going to merge and iterate on this since it will be less typing for all :smile:

**Pull request merged**